### ISSUE TITLE
[BITV] 9.4.1.2/9.3 - The status of the button "Toggle user actions menu" is only communicated to screen reader users in the "expanded" state using the aria-expanded="true" attribute. The "aria-expanded" attribute should be used in both states. (1)

### ISSUE THREAD

**Issue by JuliaKirschenheuter** [nextcloud/server#41907]
https://report.bitvtest.de/default-en/cbedb5ef-4815-4cba-a83d-995599c178c3.html#checkpoint-bfbf07e5c3-v9-n3

**Comment by ShGKme** [comment]
The title is correct. It is in German because the page is actually in German.

The login page seems to be in English because the login form is not translated.

See: #41881

**Comment by ShGKme** [comment]
Same as https://github.com/nextcloud/server/issues/41897
